### DISCUSSION TITLE
Brunch Install Step the Idea of dualboot with another linux distro using such method like in brunch

### DISCUSSION THREAD

**Discussion by msprestigepro** [sebanc/brunch#1778] [General]
Hi everyone, i have a question, is there a way to install linux distro using brunch method i mean dualboot alongside windows without shrinking disk partition and creating usb bootable? installing the linux distro into .img file like brunch chrome os does? i like the idea, is it possible? thanks in advance

**Comment by sebanc** [comment]
Hi, you can have a look at my "linuxloops" GitHub repository.

**Comment by msprestigepro** [comment]
Thank you i'll have a look

Pada tanggal Sen, 6 Feb 2023 06.07, sebanc ***@***.***>
menulis:
